db/models/review.test.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const db = require('APP/db'),
+  { Review } = db,
+  { expect } = require('chai'),
+  Promise = require('bluebird')
+
+describe('The `Review` model', () => {
+  /**
+   * First we clear the database and recreate the tables before beginning a run
+   */
+  before('Await database sync', () => db.didSync)
+
+  /**
+   * Next, we create an (un-saved!) Review instance before every spec
+   */
+  let review
+  beforeEach(() => {
+    review = Review.build({
+      title: "Funny Bike!",
+      content: "It makes weird noise every time I ride it. haha funny",
+      num_stars: 4.5
+    })
+  })
+
+  /**
+   * Also, we empty the tables after each spec
+   */
+  afterEach(function() {
+    return Promise.all([
+      Review.truncate({ cascade: true })
+    ])
+  })
+
+  describe('attributes definition', () => {
+
+    it('included `title`, `content`, `num_stars` fields', () => {
+      return Review.save()
+        .then(savedReview => {
+          expect(savedReview.title).to.equal("Funny Bike!")
+          expect(savedReview.content).to.equal("It makes weird noise every time I ride it. haha funny")
+          expect(savedReview.num_stars).to.equal(4.5)
+        })
+    })
+
+    it('requires `title`', () => {
+      review.status = null
+      return review.validate()
+        .then(result => {
+          expect(result).to.be.an.instanceOf(Error)
+          expect(result.message).to.contain('title cannot be null')
+        })
+    })
+
+
+    it('requires `content`', () => {
+
+      review.content = null
+
+      return review.validate()
+        .then(result => {
+          expect(result).to.be.an.instanceOf(Error)
+          expect(result.message).to.contain('content cannot be null')
+        })
+    })
+    it('requires `content`', () => {
+
+      review.num_stars = null
+
+      return review.validate()
+        .then(result => {
+          expect(result).to.be.an.instanceOf(Error)
+          expect(result.message).to.contain('num_stars cannot be null')
+        })
+    })
+
+    it('requires `title` to include at least one letter', () => {
```

Comment:
Does this actually test that we have the appropriate validation in place?  What happens if someone wrote a model with a validation that checked that titles was an integer.  This test would pass, but we would not be testing what we wanted to.  I will shorten this as "Does this test what we want?"